### ISSUE TITLE
old jobs purging didn't remove  directories contain jid

### ISSUE THREAD

**Issue by tjyang** [saltstack/salt#19726]
AFAIK, for default setting, JIDs older than two days will be removed. But not the directories contain the old jid itself.
Can /var/cache/salt/master/jobs/{6a|a5|73} types of folder removed when purging jids also ?

```
[root@master01 jobs]# pwd
/var/cache/salt/master/jobs
[root@master01 jobs]# find . -type d
<snip>
./bc/5e99cc15b09d1da71f68614179a108/lpdsp01
./85
./5a
./b2
./87
./e2
./7b
./7b/4f1f8d56495177ebcc7da23e9dc951
./7b/4f1f8d56495177ebcc7da23e9dc951/lnode2
./fb
./d6
./d6/b9829358393464ae00dee36a4493d3
./d6/b9829358393464ae00dee36a4493d3/lnode2
./6a
./a5
./73
./e1
./aa
./dd
./c5
[root@master01 jobs]# ls -ld 6a a5 73
drwxr-xr-x 2 root root 4096 Dec  6 16:09 6a
drwxr-xr-x 2 root root 4096 Jan  9 09:55 73
drwxr-xr-x 2 root root 4096 Jan  3 20:27 a5
[root@master01 jobs]# date
Wed Jan 14 09:20:32 EST 2015
[root@master01 jobs]# ls -ld bc
drwxr-xr-x 3 root root 4096 Jan 13 12:33 bc

```

**Comment by basepi** [comment]
Those top level directories are created to lower the load of stat-ing those directories. We have a folder for all the two-character hex combinations, 00 through ff. Those folders are created as part of startup and are never removed. Additionally, the jobs folder is not designed to be browsed manually -- you should be using the jobs runner instead, so I'm not sure why those folders are a problem.

For now I think we're going to keep those folders around.

I should also point out that jobs older than 24 hours will be deleted by default. (Not 2 days as you mentioned)

**Comment by rallytime** [comment]
This actually was fixed recently in #33328. Closing.
